lib/iris/fileformats/grib/load_rules.py

```diff
@@ -390,7 +395,9 @@ def add_bounded_time_coords(aux_coords_and_dims, grib):
             (grib.productDefinitionTemplateNumber == 1):
         aux_coords_and_dims.append((DimCoord(points=grib.perturbationNumber, long_name='ensemble_member', units='no_unit'), None))
 
-    if grib.productDefinitionTemplateNumber not in (0, 8):
+    if \
+            (grib.edition == 2) and \
+            grib.productDefinitionTemplateNumber not in (0, 8):
```

Comment:
Yes. GRIB1 doesn't have the concept of a product definition template so this only applicable to GRIB2